### DISCUSSION TITLE
foldable/collapse Hardware info in Host/hardware

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#8381] [Supervisor]
Please consider returning to the 'old' way of displaying the hardware on the system to start out with, and allow the fully opened list we now get by default, as an option per collapsible category.

Usecase: the main reason we need this window is for finding the USB port sticks are connected to . Before we scrolled down to the usb section, immediately see the 4 ports, and click 1 of these.

Currently we have pages, and they really are many, with huge amounts of hardware info, and it is a rail pain to find any USB port, let alone the correct one.

It would really be appreciated if this window could be made more approachable, use collapsible categories eg, and list a much shorter window to begin with.
thanks for considering

this is one of many scrollable pages to go through, before finally finding the section on usb, which is just as cluttered with info

<img width="586" alt="Schermafbeelding 2021-02-12 om 09 24 57" src="https://user-images.githubusercontent.com/33354141/107748566-8d781800-6d19-11eb-84f3-c06f630f4228.png">

**Comment by SmartKobol** [comment]
Dear  Mariusthvdb, This is not really an answer, rather than a question. Even in the latest 2022.6.4 where can I find any info about the hardware, specifically usb dongle plugged-in?  I have traced Setting/System session but I have been lost in the new structure without any success...

**Reply by Mariusthvdb** [reply]
Config/system/hardware and then the 3 dots menu

**Reply by Mariusthvdb** [reply]
![120B37D3-1D07-491B-B319-DB0245D82760](https://user-images.githubusercontent.com/33354141/173123413-5fffedbc-2cb8-4199-8947-c8b36c15bbe2.png)
